ui/templates/upload.html

```diff
@@ -1,5 +1,6 @@
-{% extends "base.html" %}
+{% extends "repo_base.html" %}
```

Comment:
Internet total terrible.  I just pushed a change to swap the nav to go to the listing page instead.  After this, I think @pwilkins  will have to take over